integration_test/version_test.go

```diff
@@ -36,7 +48,7 @@ func TestKrewVersion(t *testing.T) {
 		}
 		optionValue := lineSplit.Split(line, 2)
 		if len(optionValue) < 2 {
-			t.Errorf("Expect `krew version` to output `OPTION VALUE` pair separated by spaces, got: %v", optionValue)
+			test.t.Errorf("Expect `krew version` to output `OPTION VALUE` pair separated by spaces, got: %v", optionValue)
```

Comment:
Better return an error from this function and log the result in the caller, i.e.

```suggestion
return fmt.Errorf("Expect `krew version` to output `OPTION VALUE` pair separated by spaces, got: %v", optionValue)
```